.github/workflows/cd-psgallery.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: cd-psgallery
+
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
```

Comment:
I'm still getting used to Github, so the syntax is still a learning curve for me.

Do we want this workflow to run after the CI workflow? If so, shouldn't the `workflow_run` event be used here? Something like they have in their docs: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_runbranchesbranches-ignore

Reply:
Ah nevermind, I forgot we decided on a manual gate in between CI and CD.